style.css

```diff
@@ -10,17 +10,13 @@ body {
   padding: 0;
```

Comment:
- [OPTIONAL]  I'll suggest you add the smooth scroll property to your CSS so that when a user clicks on a menu item, the page scrolls smoothly to that section. :+1: :+1: You can achieve that by adding this to the top of your CSS file. `html {
  scroll-behavior: smooth;
}` :smile: